composer.json

```diff
@@ -9,6 +9,7 @@
     "spryker/event-dispatcher-extension": "^1.0.0",
     "spryker/kernel": "^3.49.0",
     "spryker/laminas": "^1.0.0",
+    "spryker/propel": "^3.28.0",
```

Comment:
For tests we need to use "dev-... as" here for them to run
Did you do that before? So we know all is good when releasing?

I wonder if we need to fix up the auto release process to include these aliasing changes for release.
So we have better testing beforehand.